sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/DiscountTag.java

```diff
@@ -0,0 +1,24 @@
+package com.exadel.sandbox.team5.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "discount_tag")
+public class DiscountTag extends BaseEntity {
```

Comment:
This class can be removed if we will use relationship many to many in class `Discount`.